ui/src/components/PendingTasksTable.tsx

```diff
@@ -345,8 +342,20 @@ function Row(props: RowProps) {
           />
         </IconButton>
       </TableCell>
-      <TableCell component="th" scope="row">
+      <TableCell component="th" scope="row" style={{width:'200px'}}>
```

Comment:
nit: Avoid inline styles if possible. Add className and specify the styles like others.

Also, don't we need to apply this style in other tables too?

Reply:
done!